rpc/src/v1/helpers/subscribers.rs

```diff
@@ -37,7 +39,7 @@ impl str::FromStr for Id {
 }
 impl Id {
 	pub fn as_string(&self) -> String {
-		format!("0x{:?}", self.0)
+		format!("{:?}", self.0)
```

Comment:
This should be changed to `serde_json::to_string_pretty(&self.0)` for performance reasons but I think it is out-of-scope of this PR and should be fixed in another one